### PR TITLE
Task CopyFilesV2 migrate to Node10

### PR DESCRIPTION
**Pull request by golubev-ivan**
**Task name**: CopyFilesV2

**Description**: Migrated task to Node10
- Execution handler is updated
- Typings are deleted
- Tests are fixed

`fs.utimes = promisify(function (targetPath, atime, mtime, err) {});` - fs.utimes currently require function as promise type.
`fs.readdirSync = (p: fs.PathLike, o?: any): any => {};` - fs.readdirSync has several overrides and for correct working needs to specify `any` type for the second parameter and output.

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** #15516

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by EzzhevNikita — APPROVED** [review]

File: Tasks/CopyFilesV2/task.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 190,
-        "Patch": 1
+        "Minor": 197,
```
Comment (anatolybolshakov) [inline Tasks/CopyFilesV2/task.json:19]: Since we've already have Courtesy Push PR prepared - could it make sense to bump it to 198?

Comment (golubev-ivan) [reply Tasks/CopyFilesV2/task.json:19]: Ok, I will bump it to 198

**Review by anatolybolshakov — APPROVED** [review]

**Pull request merged**